.wiki/Configuration.md

```diff
@@ -37,6 +42,47 @@ for [repository definitions](https://github.com/artipie/artipie/wiki/Configurati
 It locates a storage where all config files for each repository are located. Keep in mind,
 Artipie user should have read and write permissions for this storage.
 
-You may want configure it via environment variables:
+Here is the example of Artipie configuration files structure for `flat` layout based on 
```

Comment:
Think, it should go right after layout description, before storage.
Maybe, create subtitles to make description more structured? Additionally, subtitles can be used as a link from another files/sections.
### Layouts
### Storage
### Credentials
### Config

Reply:
I do not agree that configs layout examples should be before storage description because in the layout example we use path from storage configuration.

As for subtitles - there is nothing to make subtitles from, credentials and storages are properly described on other pages.

Reply:
ok